src/dialog.jsx

```diff
@@ -70,7 +78,7 @@ const TransitionItem = React.createClass({
     });
 
     setTimeout(() => {
```

Comment:
Woops, I thought I came back to fix that. I must've forgotten or lost it. Thanks!